<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ACC_FactorCompanyDialog</class>
 <widget class="QDialog" name="ACC_FactorCompanyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <property name="verticalSpacing">
    <number>3</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="gbFactorCompany">
     <property name="title">
      <string>Factor Companies</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <property name="horizontalSpacing">
       <number>3</number>
      </property>
      <property name="verticalSpacing">
       <number>1</number>
      </property>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>1</number>
        </property>
        <item>
         <widget class="QToolButton" name="pbAdd">
          <property name="minimumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Add authority</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../../db/src/gui/db.qrc">
            <normaloff>:/images/icons/add.png</normaloff>:/images/icons/add.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="pbDelete">
          <property name="minimumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Delete authority</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normalon>:/images/icons/delete.png</normalon>
           </iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="lblTvInfoAuth">
          <property name="text">
           <string>Click item to select or [F2] to edit</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QTableView" name="tvFactorCompany"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QFormLayout" name="formLayout">
     <property name="labelAlignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="horizontalSpacing">
      <number>4</number>
     </property>
     <property name="verticalSpacing">
      <number>3</number>
     </property>
     <property name="leftMargin">
      <number>3</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Factor Company Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="leFactorName">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Contact Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="leContactName"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Telephone</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="leTelephone">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Fax</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="leFax">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Email</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="leEmail"/>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QFormLayout" name="formLayout_2">
     <property name="horizontalSpacing">
      <number>4</number>
     </property>
     <property name="verticalSpacing">
      <number>3</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Address Line 1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="leAddressLine1">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Address Line 2</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="leAddressLine2"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Address Line 3</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="leAddressLine3"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Address Line 4</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="leAddressLine4"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Address Line 5</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="leAddressLine5"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Address Line 6</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="leAddressLine6">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>1</number>
     </property>
     <item>
      <widget class="QPushButton" name="pbHelp">
       <property name="text">
        <string>&amp;Help</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>158</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbOk">
       <property name="text">
        <string>&amp;OK</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tvFactorCompany</tabstop>
  <tabstop>pbAdd</tabstop>
  <tabstop>pbDelete</tabstop>
  <tabstop>leFactorName</tabstop>
  <tabstop>leContactName</tabstop>
  <tabstop>leTelephone</tabstop>
  <tabstop>leFax</tabstop>
  <tabstop>leEmail</tabstop>
  <tabstop>leAddressLine1</tabstop>
  <tabstop>leAddressLine2</tabstop>
  <tabstop>leAddressLine3</tabstop>
  <tabstop>leAddressLine4</tabstop>
  <tabstop>leAddressLine5</tabstop>
  <tabstop>leAddressLine6</tabstop>
  <tabstop>pbOk</tabstop>
  <tabstop>pbCancel</tabstop>
  <tabstop>pbHelp</tabstop>
 </tabstops>
 <resources>
  <include location="../../../db/src/gui/db.qrc"/>
 </resources>
 <connections/>
</ui>
